starter-code/scripts/articleView.js

```diff
@@ -16,50 +16,82 @@ articleView.populateFilters = function() {
   });
 };
 
+
 articleView.handleAuthorFilter = function() {
   $('#author-filter').on('change', function() {
-  if ($(this).val()) {
-      /* TODO: If the slect box changes to an option that has a value, we should:
-          1. Hide all of the articles
-          2. Fade in only the articles that match based on on the author
-            that was aselected. Hint: use an attribute selector to find
-            those articles that match the value, and then fade them in.
-        */
-  } else {
-    /* Otherwise, we should:
-        1. Show all the articles except the template */
-  }
+    if ($(this).val()) {
+      $('article').hide();
+      $(`article[data-author="${ $(this).val() }"]`).fadeIn(1000)
+      // $('article[data-author=" ' + $(this).val() + '"]').show();
+      //     /* TODO: DONE: If the slect box changes to an option that has a value, we should:
+      //         1. Hide all of the articles
+      //         2. Fade in only the articles that match based on on the author
+      //           that was aselected. Hint: use an attribute selector to find
+      //           those articles that match the value, and then fade them in.
+      //       */
+    } else {
+      $('article').not('.template').show();
```

Comment:
This is a great way of showing all of the articles that aren't the template! One thing that you might want to consider is that it's a good practice to be as specific as possible in your selector.  

If you have any other articles in your HTML that aren't related to your blog posts (but are still semantically articles), this could cause unintended behavior. Instead, you could try this:

```
$('#articles article').show();
$('#articles article.template').hide();
```